### DISCUSSION TITLE
'Files' view should show/link to which note the item is affiliated with.

### DISCUSSION THREAD

**Discussion by OddHouseGames** [standardnotes/forum#2842] [Suggestions]
It would be useful if when in 'Files' view, that the individual items show/link to the affiliated note for ease of housekeeping and item association management.

**Comment by myreli** [comment]
This is available with the new table view for Files:

<img width="1147" alt="image" src="https://user-images.githubusercontent.com/17554234/221364081-65e74196-d61e-4549-9d46-63d688c7601b.png">

You can also opt-in to this layout (beta) for any other view/tag:

<img width="1147" alt="image" src="https://user-images.githubusercontent.com/17554234/221364168-ae63ebdc-c39f-4b3a-86cb-5d04cb8f56bd.png">
